iOS/Core/PixelEvent.swift

```diff
@@ -973,6 +973,7 @@ extension Pixel {
         case privacyProAuthV2MigrationFailed
         case privacyProAuthV2MigrationSucceeded
         case privacyProAuthV2GetTokensError
+        case privacyProSubscriptionExpired
```

Comment:
WDYT about also firing pixel for `.entitlementsDidChange` notification? We could also consider sending one where the entitlements list is in params and what is even more interesting maybe a separate pixel for this notification when entitlements are empty. I would see this as a proxy for the expired subscription scenario but potentially also a signal of something not being right if it differs from the expired subscription pixel.